dotnet/src/Connectors/Connectors.AzureAIInference/Connectors.AzureAIInference.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.AzureAIInference</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);NU5104;SKEXP0001,SKEXP0070</NoWarn>
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
We should make this Beta to match [Azure.AI.Inference](https://www.nuget.org/packages/Azure.AI.Inference/1.0.0-beta.1)

`<VersionSuffix>beta.1</VersionSuffix>`